src/main/java/com/jcraft/jsch/Session.java

```diff
@@ -3198,6 +3211,12 @@ private void checkConfig(ConfigRepository.Config config, String key) {
     if (value != null)
       this.setConfig(key, value);
   }
+  
+  public void addToFingerprintList(String channelFingerprint)
+	{
+		String []fingerprints = channelFingerprint.split("~");
```

Comment:
Is there some standard you can point to that has SSH host key fingerprints concatenated together via `~` characters?
Or is this some sort of locally invented format?

Reply:
I have changed the separator to " , " now.